cscs-checks/mch/set_openacc_device.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class SetOpenaccDevice(rfm.RegressionTest):
+    def __init__(self):
+        self.descr = 'Use OpenAcc, CUDA, MPI, and C++ on multi-device nodes'
```

Comment:
I think this description is not very accurate. Also should change the test name as well to something like `MultiDeviceOpenACCTest`?

Reply:
I have changed the name of the check as you suggested @vkarak and updated the description.